plugin/arcconf/arcconf.py

```diff
@@ -468,14 +471,15 @@ def volumes(self, search_key=None, search_value=None,
                 num_lds = len(ld_infos)
                 for ld in range(num_lds):
                     ld_info = ld_infos[ld]
-                    ld_num = ld_info['logicalDriveID']
+                    ld_id = ld_info['logicalDriveID']
                     ld_name = ld_info['name']
-                    pool_id = '%s:%s' % (sys_id, ld_num)
-                    lsm_vol = \
-                        Arcconf._arcconf_ld_to_lsm_vol(ld_info, pool_id,
-                                                       sys_id, cnt,
-                                                       str(ld_num),
-                                                       ld_name)
+                    raid_level = ld_info['raidLevel']
+                    chunk_data = ld_info['Chunk']
+                    for array_id in chunk_data:
+                        consumer_array_id = array_id['consumerArrayID']  # consumerArrayID in all the chunk will be same
+                    pool_id = '%s:%s' % (sys_id, consumer_array_id)
+                    lsm_vol = Arcconf._arcconf_ld_to_lsm_vol(ld_info, pool_id, sys_id, cnt, consumer_array_id,
+                                                             str(ld_id), raid_level, ld_name)
```

Comment:
The `ld_info` is already passed to `_arcconf_ld_to_lsm_vol()`. Please just read `raid_level`, `ld_id`, `ld_name` inside `_arcconf_ld_to_lsm_vol()`.

Reply:
Done - commit id - a70a0a1a4fffd20be4239def1f0e0a2d91d1b41b